rpy2/situation.py

```diff
@@ -101,10 +102,27 @@ def r_home_from_registry() -> Optional[str]:
     # the for-loop breaks at the first hit.
     for w_hkey in [winreg.HKEY_CURRENT_USER, winreg.HKEY_LOCAL_MACHINE]:
         try:
-            with winreg.OpenKeyEx(w_hkey,
-                                  'Software\\R-core\\R',
-                                  0, winreg.KEY_QUERY_VALUE) as hkey:
-                r_home = winreg.QueryValueEx(hkey, 'InstallPath')[0]
+            with winreg.OpenKeyEx(w_hkey, 'Software\\R-core\\R') as hkey:
+
+                # >v4.x.x: grab the highest version installed
+                def get_version(i):
+                    try:
+                        return Version(winreg.EnumKey(hkey, i))
+                    except Exception:
+                        return None
+
+                latest = max(
```

Comment:
Does Windows have the notion of the default install? Something akin to the first executable with a matching name found when searching in the `PATH`.

If there is, going for the latest version might not always be what a user will expect.

Reply:
Not that I could tell. You just get multiple Start Menu shortcuts, one for different versions. And as far as I remember, adding R executable to the system path is completely left up to users.

So, I figured that `rpy2` should get the latest version by default, and user can change this behavior via `R_HOME` env var.

Reply:
Ok. Make sense then.